proto/system_config.proto

```diff
@@ -126,6 +126,9 @@ message DaqConfig {
 
   // Drop into console mode after test
   bool use_console = 52;
+
+  // Port of the server to receive device testing result
+  bool device_testing_server_port = 53;
```

Comment:
again don't use testing -- device_result_port would be fine (is server redundant, there's never a client port...?)  What do we have elsewhere where a port is specified?

Reply:
This is the only place in the config where there is a port. Changed to device_result_port.